i18n/src/main/resources/bisq_easy.properties

```diff
@@ -144,23 +143,17 @@ bisqEasy.component.amount.baseSide.tooltip.taker.offerPrice=with the offer price
 
 
 bisqEasy.tradeWizard.amount.limitInfo.overlay.headline=Reputation-based trade amount limits
-bisqEasy.tradeWizard.amount.limitInfo.overlay.close=Close overlay
+bisqEasy.tradeWizard.amount.limitInfo.overlay.learnHowToBuildReputation=Learn how to build up reputation
+bisqEasy.tradeWizard.amount.limitInfo.overlay.close=Close
 
 bisqEasy.tradeWizard.amount.seller.limitInfo=Your maximum selling amount is {0}.
 bisqEasy.tradeWizard.amount.seller.limitInfo.overlay=Your reputation score of {0} only allows you to sell {1}. \
   This is to lower security risks for buyers.\n\n\
```

Comment:
Maybe better: "Reputation from sellers provide security for buyers." or "This is to reduce security risks for buyers."

Reply:
Or we use the statement we use elsewhere with that the buyer sends fiat first...